distribution/lib/Standard/Google_Api/0.0.0-dev/src/Main.enso

```diff
@@ -58,3 +67,36 @@ initialize secret_file =
     http_transport = GoogleNetHttpTransport.newTrustedTransport
     json_factory = GsonFactory.getDefaultInstance
     Google_Api_Client.Value credential json_factory http_transport
+
+
+## PLACEHOLDER, performs google analytics call
+
+   Arguments:
+   - property_id
+   - start_date
+   - end_date
+run_google_report : Text -> Date -> Date -> Table
+run_google_report property_id:Text start_date:Date=(Date.new 2020 3 31) end_date:Date=Date.today =
```

Comment:
I believe you can ditch the type ascription above, and just add the return type ascription in-line, like so:
```suggestion
run_google_report property_id:Text start_date:Date=(Date.new 2020 3 31) end_date:Date=Date.today -> Table =
```